core/domain/email_manager.py

```diff
@@ -77,6 +77,46 @@ def log_new_error(*args, **kwargs):
         'subject': _PLACEHOLDER_SUBJECT,
         'html_body': _PLACEHOLDER_HTML_BODY,
     })
+
+_EDITOR_ROLE_EMAIL_SUBJECT = (
```

Comment:
OK -- I think we shouldn't do all this as a config property, it's getting too complicated since the email text needs to be parameterized. There's also some confusion between strings and string templates -- some of these strings won't make sense without interpolation. Here's what I suggest instead:

1) Call this _EDITOR_ROLE_EMAIL_SUBJECT_TEMPLATE and the following constant _EDITOR_ROLE_EMAIL_BODY_TEMPLATE.

2) Remove the EDITOR_ROLE_EMAIL_CONTENT template. We just won't allow editing of this template via the admin page.

3) Add a fine-grained boolean flag in feconf.py (near all the other email settings) that requires users to explicitly turn role-notification-emails functionality on. Above that flag, add a comment saying: if you want to turn this on, consider changing the email templates in email_manager.py accordingly.